commcare_connect/templates/opportunity/user_profile.html

```diff
@@ -22,55 +22,114 @@
-          <div class="col">{{access.last_visit_date}}</div>
+
+        <!-- Stats Grid -->
+        <div class="row g-3 mb-4">
+          <div class="col-md-6 col-lg-3">
+            <div class="card h-100 border-0 bg-light">
+              <div class="card-body text-center">
+                <i class="bi bi-telephone mb-2 text-primary fs-4"></i>
+                <h6 class="text-muted mb-1">{% translate "Phone" %}</h6>
+                <div>{{access.user.phone_number}}</div>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-6 col-lg-3">
+            <div class="card h-100 border-0 bg-light">
+              <div class="card-body text-center">
+                <i class="bi bi-graph-up mb-2 text-success fs-4"></i>
+                <h6 class="text-muted mb-1">{% translate "Learn Progress" %}</h6>
+                <div class="d-flex align-items-center justify-content-center">
+                  <div class="progress w-75 bg-white">
+                    <div class="progress-bar bg-success" role="progressbar"
+                         style="width: {{access.learn_progress}}%" aria-valuenow="{{access.learn_progress}}"
+                         aria-valuemin="0" aria-valuemax="100"></div>
+                  </div>
+                  <span class="ms-2">{{access.learn_progress}}%</span>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-6 col-lg-3">
+            <div class="card h-100 border-0 bg-light">
+              <div class="card-body text-center">
+                <i class="bi bi-pin-map mb-2 text-info fs-4"></i>
+                <h6 class="text-muted mb-1">{% translate "Total Visits" %}</h6>
+                <div>{{access.visit_count}}</div>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-6 col-lg-3">
+            <div class="card h-100 border-0 bg-light">
+              <div class="card-body text-center">
+                <i class="bi bi-calendar mb-2 text-warning fs-4"></i>
+                <h6 class="text-muted mb-1">{% translate "Last Visit" %}</h6>
+                <div>{{access.last_visit_date}}</div>
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
-       <div class="position-relative">
-        <div id="user-visit-map" class="mt-3 mb-2 rounded border" style="height: 300px;"></div>
-        <div id="legend" class="card position-absolute bottom-0 end-0 m-2 z-1" x-data="{ show: false}" x-show="show">
-          <div class="card-body">
-            <h6 class="card-title">Catchment Areas</h6>
-            <div class="mb-2 d-flex align-items-center">
-            <span class="d-inline-block me-2"
-                  style="width: 20px; height: 20px; opacity: 0.3; background-color: #3366ff;"></span>
-              <span>Active</span>
+
+        <!-- Map Section -->
+        <div class="position-relative">
```

Comment:
(This is not blocking, and you can followup in a separate PR.)

Are all of these divs necessary to get the desired styling, I feel like some of these could be redundant.